dipy/viz/horizon/tab/base.py

```diff
@@ -81,6 +79,26 @@ def on_slice_change(self, _x_value, _y_value, _z_value):  # noqa: B027
         changes."""
         pass
 
+    def show(*args):  # noqa: B027
+        """Show elements in the scene. This method will be implemented in TabManager.
+
+        Parameters
+        ----------
+        *args : HorizonUIElement or FURY actors
+            Elements to be hidden.
+        """
+        pass
+
+    def hide(*args):  # noqa: B027
+        """Hide elements from the scene. This method will be implemented in TabManager.
+
+        Parameters
+        ----------
+        *args : HorizonUIElement or FURY actors
+            Elements to be hidden.
+        """
+        pass
```

Comment:
not sure to understand this change? What was wrong with the previous definition ?

before we were returning `None` and now it is just a `pass`. Why not `raise NotImplementedError` ?

Can you clarify ? Thanks

Reply:
The test cases can not mock this method and they need a proper definition. This method gets updated in Tab Manager anyways for all the tabs.

We can raise the `NotImplementedError`.